android/config.json

```diff
@@ -6,5 +6,6 @@
         "FacebookSDK400/facebook/build/outputs/aar/facebook-debug.aar"
```

Comment:
since you have moved to gradle, is this still needed ?

Reply:
No, I will remove

Reply:
Removed